src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1443,7 +1446,7 @@ class ApplicationManagerImpl
    * will send TTS global properties to HMI after timeout
    */
   std::map<uint32_t, TimevalStruct> tts_global_properties_app_list_;
-
+  std::set<connection_handler::DeviceHandle> querry_apps_devices_;
```

Comment:
@Ypostolov `query` not `querry`!

Reply:
fixed cf05050f8c0ed76ade88e39c498ed2df15b78c82